manual/src/refman/extensions/alerts.etex

```diff
@@ -17,30 +17,64 @@ component is referenced).
 
 The "ocaml.alert" or "alert" attribute serves two purposes: (i) to
 mark component with an alert to be triggered when the component is
-referenced, and (ii) to control which alert names are enabled.  In the
-first form, the attribute takes an identifier possibly
+referenced, and (ii) to control which alert names are enabled.
+
+\subsection{ss:marking}{Marking component with an alert}
+
+In the first form, the attribute takes an identifier possibly
 followed by a message. Here is an example of a value declaration marked
 with an alert:
 
-\begin{verbatim}
+\begin{caml_example*}{signature}
 module U: sig
   val fork: unit -> bool
     [@@alert unix "This function is only available under Unix."]
 end
-\end{verbatim}
+\end{caml_example*}
 
 Here "unix" is the identifier for the alert.  If this alert category
 is enabled, any reference to "U.fork" will produce a message at
 compile time, which can be turned or not into a fatal error.
 
-And here is another example as a floating attribute on top
+For other structure or signature components, the syntax is as follows:
+
+\begin{caml_example*}{verbatim}
+module type Sig = sig
+  val v: int [@@alert name "message"]
+end
```

Comment:
I would also propose to add an example with a signature constraint

```ocaml
module M: sig 
   val x: int
   val y: int [@@deprecated]
 end   = struct
   let (x[@deprecated]) = 0
   let y = 1
 end
 ```
 to explain the behavior in this case.

Reply:
Ok, I added that.